aircraft/global5000/global5000.xml

```diff
@@ -0,0 +1,953 @@
+            <table>
+                <independentVar>aero/h_b-mac-ft</independentVar>
+                <tableData>
+                    0.0000	0.0480
+                    0.1000	0.5150
+                    0.1500	0.6290
+                    0.2000	0.7090
+                    0.3000	0.8150
+                    0.4000	0.8820
+                    0.5000	0.9280
+                    0.6000	0.9620
+                    0.7000	0.9880
+                    0.8000	1.0000
+                </tableData>
+            </table>
+        </function>
+ 
+	 <function name="aero/function/kCLge">
+	    <description>Change_in_lift_due_to_ground_effect</description>
+	    <table>
+		<independentVar>aero/h_b-mac-ft</independentVar>
+		<tableData>
+		    0.0000	1.2030
+		    0.1000	1.1270
+		    0.1500	1.0900
+		    0.2000	1.0730
+		    0.3000	1.0460
+		    0.4000	1.0280
+		    0.5000	1.0190
+		    0.6000	1.0130
+		    0.7000	1.0080
+		    0.8000	1.0060
+		    0.9000	1.0030
+		    1.0000	1.0020
+		    1.1000	1.0000
+		</tableData>
+	    </table>
+	</function>
+
+        <function name="aero/function/kCLsb">
+            <description>Change_in_lift_due_to_speed_brake</description>
+            <table>
+                <independentVar>fcs/speedbrake-pos-norm</independentVar>
+                <tableData>
+                    0.0000	1.0
+                    0.1000	0.85
+                </tableData>
+            </table>
+        </function>
+
+        <function name="aero/function/kCLsp">
+            <description>Change_in_lift_due_to_spoilers</description>
+            <table>
+                <independentVar>fcs/spoiler-pos-norm</independentVar>
+                <tableData>
+                    0.0000	1.0
+                    0.1000	0.6
+                </tableData>
+            </table>
+        </function>
+
+
+  <axis name="LIFT">
+
+    <function name="aero/coefficient/CLalpha">
+	<description>Lift_due_to_alpha</description>
+	<product>
+	    <property>aero/qbar-psf</property>
+	    <property>metrics/Sw-sqft</property>
+	    <property>aero/function/kCLge</property>
+	    <property>aero/function/kCLsb</property>
+	    <property>aero/function/kCLsp</property>
+	      <table>
+		  <independentVar>aero/alpha-rad</independentVar>
+		  <tableData>
+		     -0.20     -0.88
+		      0.00	0.00
+		      0.23	1.00
+		      0.46	0.80
+		  </tableData>
+	      </table>
+	</product>
+    </function>
+
+
+    <function name="aero/force/Lift_alpha">
+      <description>Lift due to alpha</description>
+      <product>
+          <property>aero/qbar-psf</property>
+          <property>metrics/Sw-sqft</property>
+          <table>
+            <independentVar lookup="row">aero/alpha-rad</independentVar>
+            <tableData>
+              -0.20 -0.880
+               0.00  0.000
+               0.23  1.000
+               0.60  0.880
+            </tableData>
+          </table>
+      </product>
+    </function>
```

Comment:
As @seanmcleod mentioned this `<function>` seems to be a duplication of the function `aero/coefficient/CLalpha` just above. The result of this duplication is that the lift will be twice as expected.
```suggestion
```

Reply:
It is my fault...one is lift an the other is supposed to be CL only...